lesson11/checkpoint11/css/main.css

```diff
@@ -0,0 +1,98 @@
+body {
+  background-color: grey;
+  font-family: Helvetica, Arial, sans-serif;
+}
+
+#about {
+  background-color: peachpuff;
+}
+
+#proficiency {
+  background-color: peachpuff;
+}
+
+.section {
+  background-color: paleturquoise;
+  border: 2px solid black;
+}
+
+.section p {
+  text-indent: 20px;
+}
+
+footer {
+  text-align: end;
+}
+
+/* 
+ * This is our main class for all anchor tags (links)
+ * which are placed in the navigation.
+ * Using a descendant selector:
+ * https://www.w3schools.com/css/css_combinators.asp
+ */
+nav a {
+  text-decoration: none;
+  color: azure;
+}
+
+/* Applying some basic text shadow over my greeting */
+#greeting {
+  text-shadow: 3px 3px 4px azure;
+  text-align: center;
+  font-family: Baskerville, "Baskerville Old Face", Garamond, "Times New Roman",
+    serif;
+}
+
+ul.main-menu {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+/* The `link` pseudo-class applies styles to unvisited links. 
+ * If you visit or have visited this link, then the `link` pseudo class
+ * is not going to get triggered anymore unless you to clear your browser history.
+ * - In our case we use the same styles for both visited and unvisited links.
+ */
+a:link, a:visited {
+  color: black;
+  padding: 10px 20px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+}
+
+/*
+ * :active MUST come after :hover (if present) in the CSS definition in order to be effective!
+ * More info at https://www.w3schools.com/cssref/sel_active.asp 
+ */
+a:hover, a:active {
+  background-color: cadetblue;
+  color: white;
+}
+
+/*
+ * Lesson: CSS Units
+ * Increaes font size of the h2 header elements in the sections
+ */
+ h2 {
+   font-size: 35px;
+}
+
+/*
+ * Lesson: CSS Units
+ * Explicitly declare the profile image size
+ */
+#profilePic {
+  width: 100%;
+  height: 100%;
+}
+
+/*
+ * Lesson: CSS Units
+ * Center the profile image container and make its size 20% relative to the screen width
+ */
+#profilePicContainer {
+  width: 20%;
+  margin: auto;
+}
```

Comment:
Extra addition to the checkpoint. Let's use max-width to show how to limit page content (using the nav as example)

```suggestion
}

.container {
  max-width: 1000px; /* or use 80% */
  margin: auto;
}

/* goal is to have the Header take up most of above-the fold view */
header {
  height: 800px;
}

```

Reply:
Setting the header property fixed to 800px doesn't look good on smaller screen sizes:
![image](https://user-images.githubusercontent.com/5216020/116006120-32607c80-a60a-11eb-9565-8480fe94dba2.png)

Reply:
Do you have a better suggestion how to do this? @impactmass

Reply:
> Setting the header property fixed to 800px doesn't look good on smaller screen sizes:

Right. I think that's ok. We have a few things (including this one) that we will change it when we get to the lesson on "Responsive Web Design". So far we haven't discussed how things should/will look on mobile. Ambitious students can already fix it if then want.

Or do you prefer we remove that line? I added it as a place to test out the height.

Reply:
Yeah, I prefer we remove it so we build towards being mobile responsive and not fixing things later on to be mobile responsive.

Reply:
We can remove it.

Reply:
> not fixing things later on to be mobile responsive

If I get this point correctly, and I apply that logic to other styles we've done, then we should remove a lot. I think we've started with desktop view already, and we will need to later do other screens. Let's discuss this outside of this PR. For now, you can remove the height